cspatients/tests.py

```diff
@@ -312,6 +312,77 @@ def test_get_view(self):
             template_name="cspatients/view.html"
         )
 
+
+class PatientViewTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(
+            username="Itai"
+        )
+        self.client.force_login(self.user)
+
+    def test_view_non_existing_patient_returns_404(self):
+        response = self.client.get("/cspatients/patient/SHFLFLF")
+        self.assertEqual(response.status_code, 404)
+        self.assertTemplateUsed(
+            response=response,
+            template_name="cspatients/patient.html"
+        )
+        self.assertInHTML(
+            "Sorry. No patient such patient was found.",
```

Comment:
Could we re-word this? I think it should be "Sorry. No such patient was found."

Reply:
Yes it should.